frontend/src/features/sketches/SketchCreateForm/useSketchCreateForm.ts

```diff
@@ -25,14 +25,22 @@ export const useSketchCreateForm = ({ canvasRef }: Args) => {
   };
 
   const createSketch = async () => {
+    const showAlert = () => console.log("投稿に失敗しました");
```

Comment:
[imo]
ここのアラートなんだけど、ログイン・新規登録画面同様にwindow.alertを使った方がユーザー目線でエラーが分かりやすいから親切だと思う！
```suggestion
    const showAlert = () => window.alert("投稿に失敗しました");
```
https://github.com/givery-bootcamp/dena-2024-team1/blob/f6d5347c5d3b334e858831a2c96ae352281a0cb0/frontend/src/pages/SignUpPage.tsx#L21